catalog.html

```diff
@@ -0,0 +1,233 @@
+                  <input class="control-input" type="checkbox" name="parking" checked>Парковка
+                </label>
+              </li>
+              <li class="catalog-filter-title">
+                <label class="control">
+                  <input class="control-input" type="checkbox" name="Wi-Fi">Wi-Fi
+                </label>
+              </li>
+            </ul>
+          </fieldset>
+          <fieldset class="catalog-filter-group">
+          <legend class="catalog-filter-title">Тип Жилья:</legend>
+          <ul class="catalog-filter-list">
+            <li class="catalog-filter-title">
+              <label class="control">
+                <input class="control-input" type="radio" name="catalog-group" value="hotel" checked>Гостиница
+              </label>
+            </li>
+            <li class="catalog-filter-title">
+              <label class="control">
+                <input class="control-input" type="radio" name="catalog-group" value="motel" >Мотель
+              </label>
+            </li>
+            <li class="catalog-filter-title">
+              <label class="control">
+                <input class="control-input" type="radio" name="catalog-group" value="apartments">Апартаменты
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+        <fieldset class="catalog-filter-group">
+          <legend class="catalog-filter-title">Стоимость, ₽:</legend>
+          <label class="control">от<input class="control-input" type="text" value="0"></label>
+          <label class="control">до<input class="control-input" type="text" value="9000"></label>
+          <label class="control"><input class="control-input" type="range" min="0" max="9500"></label>
+        </fieldset>
+          <button class="button" type="submit">Применить</button>
+          <button class="button" type="reset">Сбросить</button>
+        </form>
+      </section>
+      <section class="product-card-select-section">
+        <span class="select-section-results">Найдено гостиниц: 38</span>
+        <select class="select-section-control" aria-label="Сортировка">
+          <option value="popular">Сначала дешёвые</option>
+          <option value="cheap">Сначала дорогие</option>
+          <option value="expensive">Сначала популярные</option>
+        </select>
+          <a href="#" class="button button-catalog">
+            <span class="visually-hidden">Показать карточки</span>
+          </a>
+          <a href="#" class="button button-catalog">
+            <span class="visually-hidden">Показать одну карточку</span>
+          </a>
+          <a href="#" class="button button-catalog">
+            <span class="visually-hidden">Показать список</span>
+          </a>
+      <ul class="product-list">
+        <li class="product-card">
+        <a class="product-card-link">
+          <img class="product-card-image" src="#" alt="#">
+          <h3 class="product-card-title">Amara Resort & Spa</h3>
+        </a>
```

Comment:
Формально у тебя не ошибка, но так будет лучше:
```suggestion
        <a class="product-card-link">
          <img class="product-card-image" src="#" alt="#">
        </a>
        <h3 class="product-card-title"><a>Amara Resort & Spa</a></h3>
```

Reply:
почему название нужно обернуть в тег <а>? он же тут по сути без класса и без ссылки

Reply:
Вместо ссылки будет заглушка (`href="#"`, пропустил её случайно). Класс, если будет надо, задашь.
Функционал может быть разный у ссылок. Может по клику по фотке будет отображаться увеличенное фото, а по клику по названию будет перекидывать на страницу отеля.
Требований у нас нет, поэтому и написал, что формально тут ошибки нет. Просто мой вариант будет гибче в будущем.